### PR TITLE
Adds configurable header tabs icons

### PR DESCRIPTION
**Pull request by dianabarsan**
# Description

Adds the `header_tabs` section in `app_settings` that can be used to configure which icons can be used by the app header tabs. Icons can be resource icons or FontAwesome icons, with resources taking precedence. 
Changes `resourceIcon` filter to accept a FontAwesome placeholder that is displayed if the resource icon is not found. 
Adds a new tab under the Images page in the Admin Console where header tabs icons can be configured. Resource icons are filtered to only display `svg`s. 

medic/cht-core#6410

# Code review items

- Readable: Concise, well named, follows the [style guide](https://github.com/medic/medic-docs/blob/master/development/style-guide.md), documented if necessary.
- Documented: Configuration and user documentation on [medic-docs](https://github.com/medic/medic-docs/)
- Tested: Unit and/or e2e where appropriate
- Internationalised: All user facing text
- Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in Changes.md.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by dianabarsan** [comment]
hey @garethbowen . please review along with https://github.com/medic/cht-docs/pull/161
Thanks for your time!

File: webapp/src/js/directives/header.js [hunk 1]
```diff
@@ -52,22 +54,15 @@ angular.module('inboxDirectives').directive('mmHeader', function() {
...
-      {name:'contacts',state:'contacts.detail',icon:'fa-user',translation:'Contacts',
-        permissions:['can_view_contacts','can_view_contacts_tab']},
-      {name:'analytics',state:'analytics',icon:'fa-bar-chart-o',translation:'Analytics',
-        permissions:['can_view_analytics','can_view_analytics_tab']}];
 
-      $q.all(tabs.map(tab => Auth.has(tab.permissions))).then(results => {
-        ctrl.permittedTabs = tabs.filter((tab,index) => results[index]);
-      });
 
-      ctrl.setMinimalTabs(tabs.length > 3);
```
Comment (dianabarsan) [inline webapp/src/js/directives/header.js:70]: Apparently, this never worked: https://medic.slack.com/archives/C024KTGRW/p1591305870027500

**Comment by garethbowen** [comment]
I am unable to get to this in the near future. @SCdF Would mind having a look?

**Comment by dianabarsan** [comment]
@SCdF I pushed a change that fixes the bug you outlined. 
I'm basically displaying both icons, below one another (setting them as blocks ), setting the visible area size to only show one (the top one). If the configured one (the top one) doesn't exist, there'll be nothing to be displayed and the bottom one (the default) will show.

**Comment by dianabarsan** [comment]
Tagging @garethbowen to approve or reject my CSS hack for falling back to the default icon if the configured one doesn't exist. 
If you don't mind, please have a look, Gareth. Thanks in advance.

**Comment by dianabarsan** [comment]
The screenshot that @SCdF posted, with the rocket icon, made me remember a joke, so I'm sharing it: 

This guy buys a Ferrari. Two weeks later, his manual gear box breaks and he sends it to be fixed. Two weeks after that,
his gear box breaks again... so the dealer sends a rep to "review" the guys driving technique. 
They're on the motorway, he shifts 1st 2nd 3rd .... 6th, they're at over 200km/h. The guy asks the rep:
Guy: Are you nervous?
Rep: Not at all, I know what this car can do. 
Guy: Wait till I shift into "R" for Rocket!

**Pull request merged**